index.html

```diff
@@ -0,0 +1,1117 @@
+            alt="" class="w-25 image">
+          <p class="w-75 pl-lg-4 p-side-bar" id="sports">'Friday Night SmackDown' Results (4/3): John Cena Returns
+          </p>
+        </div>
+        <div class="d-flex pt-2 pb-2 jared-margin">
+          <img
+            src="https://d.newsweek.com/en/full/1579035/wwe-wrestlemania-36-poster.webp?w=95&h=95&f=8caa604efb6871e8a8a08643ccc8f710"
+            alt="" class="w-25 image">
+          <p class="w-75 pl-4 p-side-bar" id="sportss">When and How to Watch WrestleMania 36</p>
+        </div>
+        <div class="d-flex mt-2">
+          <img src="https://d.newsweek.com/en/full/1578230/nba-2k20.webp?w=95&h=95&f=0cd041cbdf03375000c8038bd5858fed"
+            alt="" class="w-25 image">
+          <p class="w-75 pl-lg-4 p-side-bar" id="sportsss">NBA 2K Tournament: Which Players Are Taking Part, Where to
+            Watch on TV</p>
+        </div>
+      </div>
+    </section>
+    <div class="row border-top mt-lg-5 justify-content-between">
+      <section id="subscribe" class="col-lg-7 mr-lg-2">
+        <h2 class="col-lg-12 h2-bottom">SUBSCRIBE</h2>
+        <div class="row greyish pt-3 pb-3">
+          <div class="col-md-4">
+            <img
+              src="https://d.newsweek.com/en/full/1576902/april-03-2020-cover.jpg?w=170&q=95&f=e7b6af0ef76e162727acd0141bfc2e8a"
+              alt="" class="w-100">
+          </div>
+          <div class="col-md-8 pt-5 d-none d-md-block">
+            <h3>In-depth Stories You Don't See Elsewhere About Topics You Want to Read!</h3>
+            <p>All Access + Weekly Delivery.<br>
+              Choose A Membership That's Perfect for You!</p>
+            <button class="btn btn-primary subscribe p-2 rounded-0 border" type="submit">SUBSCRIBE ></button>
+          </div>
+          <div class="col-md-8 pt-5 d-md-none d-flex flex-column justify-content-center align-items-center">
+            <h3>In-depth Stories You Don't See Elsewhere About Topics You Want to Read!</h3>
+            <p>All Access + Weekly Delivery.<br>
+              Choose A Membership That's Perfect for You!</p>
+            <button class="btn btn-primary subscribe p-2 rounded-0 border" id="mobile-btn"
+              type="submit">SUBSCRIBE</button>
+          </div>
+        </div>
+      </section>
+      <section id="newsletter" class="col-lg-4 ml-lg-2 pb-3">
+        <h2 class="col-lg-12 h2-bottom">NEWSLETTER</h2>
+        <div class="row border">
+          <div class="col-md-6 newsletter-img">
+          </div>
+          <div class="col-md-6 pt-5 d-flex flex-column align-items-center">
+            <h3>The Starting 5</h3>
+            <p id="newsletter-p">See why nearly a quarter of a million subscribers begin their day with the Starting
+              5.</p>
+            <button class="btn btn-primary subscribe p-2 rounded-0 p-0 border" type="submit" id="newsletter-btn">SIGN
+              UP ></button>
+          </div>
+        </div>
+      </section>
```

Comment:
Your subscribe and newsletter sections need some work: 
- Check the titles, they should be aligned left.
- Check the spacing between them, they should be closer.
- Check the Subscribe background-color: it's different.
- Add the two images to the subscribe section.

Yours:
![Screenshot from 2020-04-13 08-18-43](https://user-images.githubusercontent.com/51926937/79123424-bf1dfc80-7d5f-11ea-98fe-0320be9037d3.png)

Original:
![Screenshot from 2020-04-13 08-18-49](https://user-images.githubusercontent.com/51926937/79123430-c6450a80-7d5f-11ea-94c3-19fad988bbbc.png)

The mobile version is not working correctly. Your Subscribe has a very large image that shouldn't have that size, and the newsletter looks like this:
![Screenshot from 2020-04-13 08-25-37](https://user-images.githubusercontent.com/51926937/79123729-68fd8900-7d60-11ea-97f6-d0c9d68bd6d4.png)